### PR TITLE
Fixed #27216 -- Corrected import location in admin deprecation messages.

### PR DESCRIPTION
**Pull request by deltaskelta**
Fixed the referenced deprecation messages

### CODE REVIEW

File: django/contrib/admin/helpers.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def contents(self):
                         if getattr(attr, "allow_tags", False):
                             warnings.warn(
                                 "Deprecated allow_tags attribute used on %s. "
-                                "Use django.utils.safestring.format_html(), "
+                                "Use django.utils.html.format_html(), "
                                 "format_html_join(), or mark_safe() instead." % attr,
```
Comment (timgraham) [inline django/contrib/admin/helpers.py]: django.utils.safestring.mark_safe()

Comment (deltaskelta) [inline django/contrib/admin/helpers.py]: Oh yes of course

**Comment by timgraham** [comment]
buildbot, add to whitelist.

File: django/contrib/admin/helpers.py (hunk 1 above)
Comment (claudep) [inline django/contrib/admin/helpers.py]: Yes, please, and take into account the pull request title changed by Tim.

**Pull request merged**